packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -1924,6 +1929,7 @@ class _SelectionHandleOverlayState extends State<_SelectionHandleOverlay> with S
                 (PanGestureRecognizer instance) {
                   instance
                     ..dragStartBehavior = widget.dragStartBehavior
+                    ..gestureSettings = eagerlyAcceptDragWhenCollapsed ? const DeviceGestureSettings(touchSlop: 1.0) : null
```

Comment:
Why 1.0? Could it be competing-slop-value minus one, or is 1.0 better? (I know you said you're still thinking about cleaning this up.)

Reply:
I like the idea of competing-slop-value minus one, or maybe minus something higher than 1 to be sure that it always wins.